test/services/atsData/PayeAtsTestData.scala

```diff
@@ -488,16 +488,16 @@ object PayeAtsTestData {
   )
 
   val expectedPercentageOrder: List[BigDecimal] =
-    List(23.5, 20.2, 12.8, 11.8, 5.3, 5.1, 4.3, 4.3, 3.6, 2.1, 1.6, 1.5, 1.5, 1.2, 1)
+    List(23.5, 20.2, 12.8, 11.8, 5.3, 5.3, 4.3, 4.3, 3.6, 2.1, 1.6, 1.5, 1.5, 1.2, 1)
 
   val expectedCategoryOrder: List[String] =
```

Comment:
This is only the expectedCategoryOrder for 2020, are we missing one for 2019 and a test for it as well?